lib/game/game_manager.ex

```diff
@@ -21,6 +21,22 @@ defmodule Reactor.GameManager do
     GenServer.call(@name, {:get_games})
   end
 
+  def get_users(game_id) do
+    game_id
+    |> to_reg
+    |> GenServer.call({:get_users})
+  end
+
+  def add_user_to_game(game_id, user) do
+    game_id
+    |> to_reg
+    |> GenServer.cast({:add_user, user})
```

Comment:
It's like I just want to add the user and then tell everyone these are all the users!